<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>901</width>
     <height>581</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>911</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>0</y>
       <width>20</width>
       <height>591</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_C1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_C2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_C3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_C4">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>310</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_C1Z">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>在线</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_C1IP">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>IP:NULL</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_C1PORT">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>端口:NULL</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pbn_C1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>断开</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbn_C2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>220</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>断开</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_C2IP">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>100</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>IP:NULL</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_C2PORT">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>160</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>端口:NULL</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_C2Z">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>40</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>离线</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pbn_C3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>510</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>断开</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_C3IP">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>390</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>IP:NULL</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_C3PORT">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>450</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>端口:NULL</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_C3Z">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>330</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>离线</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_C4PORT">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>450</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>端口:NULL</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_C4Z">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>330</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>离线</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_C4IP">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>390</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>IP:NULL</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pbn_C4">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>510</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>断开</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>281</width>
       <height>371</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>源</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>241</width>
        <height>331</height>
       </rect>
      </property>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>130</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_6">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>200</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_7">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>270</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="pbn_Send">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>470</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>60</y>
       <width>281</width>
       <height>371</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>目的</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>241</width>
        <height>331</height>
       </rect>
      </property>
      <widget class="QRadioButton" name="radioButton_13">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_14">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_15">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>130</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_16">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>200</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_17">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>270</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="maxCount">
      <number>10</number>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>客户端</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>181</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="title">
    <string>状态输出</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbn_clear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>171</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="title">
    <string>控制面板</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QPushButton" name="pbn_Manage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>客户端管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbn_Work">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>工作</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbn_Route">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>路线规划</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbn_Run">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>开启端口</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
